website/settings/defaults.py

```diff
@@ -638,6 +640,10 @@ class CeleryConfig:
                 'task': 'management.commands.update_institution_project_counts',
                 'schedule': crontab(minute=0, hour=9), # Daily 05:00 a.m. EDT
             },
+            'update_storage_usage': {
+                'task': 'management.commands.update_storage_usage',
+                'schedule': crontab(minute=0, hour=4), # Daily 0:00 a.m. EDT
```

Comment:
We might want to run this throughout the day, so it's not catching everything all at once. Might be worth checking with Matt about what a good strategy would be for this.

Reply:
It's pretty well optimized as it is, as near as we can tell. The goal right now is to need to call it less frequently per node and space out when it gets called enough that we don't have too many calls going on at a time. This may mean we call it once an hour but set the timeout to 1 week, or some similar strategy.

Reply:
The SQL hasn't changed since it was bottlenecking celery, but it may be the case that that particular issue wouldn't present with reduced recalculation frequency. Hard to say for sure.

Either way, I suspect we'll also need to do something about the fact that nodes without recent (file) changes would never display their storage usage